hazelcast/src/main/java/com/hazelcast/spi/impl/eventservice/impl/Registration.java

```diff
@@ -120,6 +130,9 @@ public void readData(ObjectDataInput in) throws IOException {
         topic = in.readString();
         subscriber = in.readObject();
         filter = in.readObject();
+        if (in.getVersion().isGreaterOrEqual(V5_3)) {
```

Comment:
FTR amended in 5cdb95d77b20150df099b3e5bcb291d87a7dbaaa, reverted in 2426fac71065446e1f583c39afdf58dc29e0f7ae.